admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -157,6 +157,31 @@ export default class OrderProductEdit {
     this.setupListener();
   }
 
+  handleEditProductWithConfirmationModal(event) {
+    const currentPriceTaxExclRaw = $(`#orderProduct_${this.orderDetailId} ${OrderViewPageMap.productEditBtn}`).data('product-price-tax-excl');
```

Comment:
Actually from this PR, same products in an order will always have the same price, since all same products are systematically updated with the data of the one being updated, so this should not be problematic.